CMake/CMakeLibs.cmake

```diff
@@ -15,7 +15,7 @@ endmacro(SET_OSQUERY_COMPILE)
 
 macro(ADD_OSQUERY_PYTHON_TEST TEST_NAME SOURCE)
   add_test(NAME python_${TEST_NAME}
-    COMMAND python "${CMAKE_SOURCE_DIR}/tools/tests/${SOURCE}" --build "${CMAKE_BINARY_DIR}"
+    COMMAND python2 "${CMAKE_SOURCE_DIR}/tools/tests/${SOURCE}" --build "${CMAKE_BINARY_DIR}"
```

Comment:
I could see this messing up if your pip is pip3, and you install our python requirements into 3, then we call python2 = missing libs. Though the thrift version we're using has string/bytes problems in python3 so we're more or less py2 bound.